comments/classified.csv

```diff
@@ -5151,3 +5151,4 @@ Your time is important to us,0,0.5
 Your use of descriptive commit messages helps in understanding the code changes clearly.,0,0.5
 Your way of thinking is corrupted,1,0.5
 your:,0,0.5
+10, however that [PR][0] is,0,0.5
```

Comment:
Ok, if you added a `,` to a CSV file, you need to surround the text portion with `"` or it's not valid CSV.

Additionally, don't put "exactly the same phrase", make it slightly different -- but enough to make the test case pass.